### DISCUSSION TITLE
Validatio Rule not found in fields.json

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#494] [Q&A] [answered]
Hi Lab !
I am using sdk 1.7.
CVS Validation found an error on this rule : BR-BT-00736-0104. I did not find this rule in fields.json
Is it normal ? 

id : BR-BT-00736-0104
location : /pin:PriorInformationNotice/cac:ProcurementProjectLot[1]/cac:TenderingTerms/cac:ContractExecutionRequirement[3]/cbc:ExecutionRequirementCode
failedAssertText : BT-736-Lot must contain a value from codelist reserved-execution

Bye
Renato

**Comment by bertrand-lorentz** [comment, marked as answer]
The fact that the field BT-736-Lot must contain a value from the codelist "reserved-execution" is indicated in fields.json, under the "codelist" property for this field.
The schematron rule BR-BT-00736-0104 corresponds to this, and enforces this constraint.

The rule identifier is currently not indicated in fields.json for various types of rules. Would it be useful for you to add those identifiers ?

**Reply by rkottmann** [reply]
yes, it would be useful to have the rule identifier.
